Try linux-aarch64 build on cirun

zklaus opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

regro-cf-autotick-bot committed:
Rebuild for libxml2 2.14

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.16.07.18.0

Commit:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.11.09.58.4

zklaus committed:
Remove os version override.

zklaus committed:
Update url

zklaus committed:
Minor fix

zklaus committed:
Move from old cdt to new conda-forge gl packaging

zklaus committed:
Remove superfluous yum_requirements

zklaus committed:
Introduce single threaded build step

zklaus committed:
Add test dependency

zklaus committed:
Add const fix patch

zklaus committed:
Merge remote-tracking branch 'upstream/main' into rebuild-libxml2214-0-1_h23269f

Commit:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.09.07.52.5

Commit:
Enable cirun-openstack-cpu-medium using Cirun

zklaus committed:
Remove timeout for linux and minor formatting

zklaus committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.10.08.45.35

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

zklaus committed:
Try to work around download restricting firewall

Commit:
Enable cirun-openstack-cpu-xlarge using Cirun

zklaus committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.10.08.45.35

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

zklaus committed:
Add syncqt.pl to create headers

zklaus committed:
Switch linux_aarch64 to cirun

zklaus committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.10.16.41.21

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

zklaus committed:
Set build platform for linux_aarch64

zklaus committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.12.06.07.39

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

zklaus committed:
Use cross compiling patches also for aarch64

zklaus committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.12.10.31.43

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

zklaus committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.12.10.31.43

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

zklaus committed:
Fix typo

zklaus committed:
Build only aarch64

zklaus committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.14.07.19.02

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

zklaus committed:
Simplify build script and make qmake available

zklaus committed:
Modify build script
